.github/workflows/docker.yaml

```diff
@@ -32,6 +32,12 @@ jobs:
             - name: Run tests
               run: npm run ci:test
 
+            - name: Publish coveralls report
```

Comment:
I had to add the coveralls build step to the `docker.yaml` file as well, because right now we're using it to build the app.